x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/reducer.ts

```diff
@@ -138,7 +148,24 @@ export const endpointListReducer: StateReducer = (state = initialEndpointPageSta
       },
     };
   } else if (action.type === 'appRequestedEndpointActivityLog') {
-    const pageData = {
+    const paging = {
+      disabled: state.endpointDetails.activityLog.paging.disabled,
+      page: action.payload.page,
+      pageSize: action.payload.pageSize,
+    };
+    return {
+      ...state,
+      endpointDetails: {
+        ...state.endpointDetails!,
+        activityLog: {
+          ...state.endpointDetails.activityLog,
+          paging: { ...paging },
```

Comment:
I don't think you need destructuring here, just `paging,` should be enough

Reply:
Quite right. I forgot to clean this up.

Reply:
done 9751707f5be89bd55ec4daeea43e17b8cffa29e4